WordPress/Classes/ViewRelated/Blog/My Site/NoSitesView.swift

```diff
@@ -0,0 +1,188 @@
+import SwiftUI
+
+protocol NoSitesViewDelegate: AnyObject {
+    func didTapAccountAndSettingsButton()
+}
+
+struct AddNewSiteConfiguration {
+    let canCreateWPComSite: Bool
+    let canAddSelfHostedSite: Bool
+    let launchSiteCreation: () -> Void
+    let launchLoginForSelfHostedSite: () -> Void
+}
+
+struct NoSitesView: View {
+
+    @SwiftUI.Environment(\.colorScheme) var colorScheme
+
+    @State private var isShowingDialog = false
+
+    let viewModel: NoSitesViewModel
+    let addNewSiteConfiguration: AddNewSiteConfiguration
+
+    weak var delegate: NoSitesViewDelegate?
+
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            Color(uiColor: .listBackground)
+                .edgesIgnoringSafeArea(.all)
+
+            makeMainView()
+                .padding(.horizontal, 32)
+
+            if viewModel.isShowingAccountAndSettings {
+                makeAccountAndSettingsButton()
+                    .padding(.horizontal, 32)
+                    .padding(.bottom, 20)
+            }
+        }
+        .frame(maxWidth: .infinity, maxHeight: .infinity)
+    }
+
+    private func makeMainView() -> some View {
```

Comment:
This suggestion/question is quite trivial but just to see what you think about it going forward:

Since in ViewBuilders, we create views as follows:
```Swift
var body: some View {
  firstView
  secondView
}
```

I find it more in line with SwiftUI syntax to use computed properties instead of functions when we can. With functions we need the function call `firstView()` instead of the syntax`firstView`.

Doesn't change anything and it won't work when you need arguments, but since these functions require no arguments I think vars would work nicer here. Wdyt?

Reply:
Done! [765149b](https://github.com/wordpress-mobile/WordPress-iOS/pull/21456/commits/765149bdea96913ebf995f5636d62af9b2325ef4)